scripts/prepare-test-data.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# A script to download and extract test data.
+#
+# Skips data that is already present in the directory.
+#
+
+echo "Begin preparing data."
+
+# Change directory to the `test` folder.
+cd "$(dirname "$0")/../test"
+
+# Extract saco dataset.
+if [ -d soco ]; then
```

Comment:
`soco` isn't special; we could put that into `data` (and have the tests use that spelling) & then everything would be in `data`, one-stop shopping